dataset/get_data.bash

```diff
@@ -2,30 +2,61 @@
 # All rights reserved.
 #
 # This source code is licensed under the license found in the
-# LICENSE file in the root directory of this source tree. 
+# LICENSE file in the root directory of this source tree.
 #
 
 preprocess_exec=./tokenizer.sed
 
+SUMGLOVE="2ffafcc9f9ae46fc8c95f32372976137"
+SUMSNLI="981c3df556bbaea3f17f752456d0088c"
+SUMMNLI="d30957eb9e172d0d32aa59cdb197d662"
+
 SNLI='https://nlp.stanford.edu/projects/snli/snli_1.0.zip'
 MultiNLI='https://www.nyu.edu/projects/bowman/multinli/multinli_0.9.zip'
 glovepath='http://nlp.stanford.edu/data/glove.840B.300d.zip'
 
+ZIPTOOL="unzip"
+SUMTOOL="md5sum"
+
+if [ "$OSTYPE" == "OSX" ]; then
```

Comment:
Did you check that $OSTYPE is actually "OSX" on your machine?
It seems to be "darwin*" for MACs.
https://stackoverflow.com/questions/394230/detect-the-os-from-a-bash-script